docs/content/2.components/CldImage.md

```diff
@@ -0,0 +1,191 @@
+# CldImage.vue
+
+---
+
+## Usage
+
+Using `CldImage.vue` component is really straight forward. It accepts the same attributes as native img tag (such us sizes, width, loading, etc) and bunch of Cloudinary specific props used to optimize the image (such as removeBackground, overlays, gravity, etc). Thanks to the auto import feature of Nuxt, you do not need to import anything. Just add `<CldImage />` to you template and optimize your images!
+
+```html
+<CldImage
+  width="987"
+  height="987"
+  :src="'images/woman-headphones'"
+  sizes="50vw"
+  crop="thumb"
+  gravity="faces"
+  removeBackground
+  tint="40:253f8c"
+  :underlay="'images/city-skyline'"
+  :overlays="[
+    {
+      position: {
+        gravity: 'north',
+        y: 60
+      },
+      text: {
+        color: 'rgb:52a4ff80',
+        fontFamily: 'Source Sans Pro',
+        fontSize: 320,
+        fontWeight: 'black',
+        text: 'MUSIC',
+        letterSpacing: -10,
+        lineSpacing: -100,
+        stroke: true,
+        border: '20px_solid_rgb:2d0eff99',
+      }
+    },
+    {
+      position: {
+        gravity: 'south',
+        y: 60
+      },
+      text: {
+        color: 'rgb:52a4ff80',
+        fontFamily: 'Source Sans Pro',
+        fontSize: 320,
+        fontWeight: 'black',
+        text: 'IS LIFE',
+        letterSpacing: -10,
+        lineSpacing: -100,
+        stroke: true,
+        border: '20px_solid_rgb:2d0eff99',
+      }
+    }
+  ]"
+/>
+```
+
+
+## Configuration
+
+Configuration for CldImage is the same as [getCldImageUrl](/helpers/getcldimageurl/configuration), which both use the same underlying API.
```

Comment:
probably need to update the getcldimageurl reference since its different here right?

Reply:
You are right. I was copying this at the airport :D